extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/cors/CORSConfig.java

```diff
@@ -61,6 +61,14 @@
     @ConfigItem
     public Optional<Duration> accessControlMaxAge;
 
+    /**
+     * The `Access-Control-Allow-Credentials` header is used to tell the
+     * browsers to expose the response to front-end JavaScript code when
+     * the request’s credentials mode Request.credentials is “include”.
+     */
+    @ConfigItem(defaultValueDocumentation = "Default value is true when origins is present and does not include '*', else it is false")
```

Comment:
I'm trying to figure out how to change either the docs or the `defaultValueDocumentation` and I'm getting confused for the 3rd time :-). 
OK, so, if this property is false (which is a default value of this property) then `Access-Control-Allow-Credentials` is false unless a matching non-wildcard Origins is present.
It feels it would be less confusing if we remove this `defaultValueDocumentation` and instead add to the javaDocs:
```
The value of this header will be set to `true` if either this property is set to `true` or a matching `Origin` which does not include `*` is present.
```
How does it sound ?

Reply:
Its clear, sounds good. Done